matrix_sdk_base/src/models/room.rs

```diff
@@ -247,6 +250,11 @@ impl Room {
         self.encrypted
     }
 
+    /// Get the resolved display name for a member of this room.
+    pub fn resolved_display_name(&self, id: &UserId) -> Option<&str> {
```

Comment:
Can we name this to be something like `user_display_name()` or `member_display_name()`?

Reply:
And drop the `resolve` part?

Reply:
Yeah.

Reply:
Done in b98d2388db0129d80f3f238c0e8632be93bd6929